step-certificates/templates/secrets.yaml

```diff
@@ -14,3 +14,30 @@ metadata:
   name: {{ include "step-certificates.fullname" . }}-provisioner-password
   namespace: {{ .Release.Namespace }}
 {{- end }}
+---
+{{- if .Values.inject.secrets.enabled }}
+apiVersion: v1
+kind: Secret
+type: smallstep.com/ca-password
+metadata:
+  name: {{ include "step-certificates.fullname" . }}-ca-password
+  namespace: {{ .Release.Namespace }}
+data:
+  password: {{ .Values.inject.secrets.ca_password }}
+---
+apiVersion: v1
+kind: Secret
+type: smallstep.com/private_keys
```

Comment:
To be consistent the standard types, we should use `-` instead of `_`.

Reply:
This should be easy to change.